src/main/java/ab/common/lib/register/RecipeListAB.java

```diff
@@ -24,6 +27,7 @@
 import thaumcraft.api.research.ResearchCategories;
 import thaumcraft.api.research.ResearchItem;
 import thaumcraft.api.research.ResearchPage;
+import thaumcraft.api.wands.*;
```

Comment:
wildcard imports are dangerous, please configure your ide to not use them, explicit imports are way better.